### PR TITLE
Delete mysql-metrics Cron Job

### PR DESCRIPTION
**Pull request by sureshc**
We no longer need the custom CloudWatch Metrics published by the `mysql-metrics` cron job because RDS Aurora MySQL. publishes most of these now (or equivalent metrics) and also RDS Performance Insights provides better analysis of database performance.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: bin/cron/mysql-metrics [hunk 1]
```diff
@@ -1,377 +0,0 @@
...
-abort 'Script already running' unless only_one_running?(__FILE__)
-
-require_relative '../../deployment'
-require 'cdo/db'
-require 'aws-sdk-cloudwatch'
-
-# Report details MySQL metrics to CloudWatch.
-# Intended to be run at 1-minute intervals.
-
-STATUS_CACHE = pegasus_dir('cache', 'mysql-status-cache.json')
```
Comment (Hamms) [inline bin/cron/mysql-metrics:12]: Looks like we can remove this line from this directory's gitignore: https://github.com/code-dot-org/code-dot-org/blob/b01bde73c4b4bddde9b61543a5c02e1a4b6c04c0/pegasus/cache/.gitignore#L13

**Review by Hamms — APPROVED** [review]
One nit, otherwise LGTM!

File: bin/cron/mysql-metrics (hunk 1 above)
Comment (sureshc) [reply bin/cron/mysql-metrics:12]: Nice catch (cache?)! Should we automate deletion of this file? When it's no longer ignored, will it get auto-committed by our staging and levelbuilder content push jobs?

Comment (Hamms) [reply bin/cron/mysql-metrics:12]: Probably worth verifying, but I think those jobs are smart enough to only commit content from specific subdirectories. We should plan to clean it up anyway, though; no preferences on whether we do so manually or automatically.

**Pull request merged**